docker-compose.yml

```diff
@@ -147,11 +147,10 @@ services:
 
   db:
     container_name: db
-    build:
-      context: database
-      dockerfile: Dockerfile
+    image: postgres:15-alpine
     volumes:
       - postgres:/var/lib/postgresql/data
+      - ./services/database:/docker-entrypoint-initdb.d/
```

Comment:
Mounting the init script like that will not change the fact that the database has to be removed before.

Reply:
yes, but the database is removed with `just wipe` - the Docker image was not.

Reply:
Doesn't just wipe rub docker-compose down -v?

Reply:
yes, it runs it, but it does not rebuild the Docker *image*, which was necessary in our old method.

Reply:
🤔 did it work for you? or did you end up manually using `psql` to add the table? or force rebuild of the `Dockerfile` ?

Reply:
hmm.. I guess i did run docker-compose up --build -d at some point.